install_config/upgrades.adoc

```diff
@@ -494,8 +636,7 @@ build of those applications after importing the new images using `oc start-build
 ====
 
 [[additional-instructions-per-release]]
-
-== Additional Manual Steps Per Release
+=== Additional Manual Steps Per Release
```

Comment:
I adjusted this to become a subsection of the overall manual steps, because it seemed like these steps are performed by the upgrade playbooks, or at least the 3.0 to 3.1 one. Is this accurate also for the Origin upgrade paths that are also still below (1.0.4 and 1.0.5)?